plugins/GSdx/GSWnd.h

```diff
@@ -60,19 +60,20 @@ class GSWndGL : public GSWnd
 {
 protected:
 	bool m_ctx_attached;
-	int m_vsync;
+	std::atomic_bool m_vsync_change_requested;
```

Comment:
``std::atomic_bool`` is just a type definition of ``std::atomic<bool>``. He has probably used this for the same reason on why we're using ``GSVector2i`` over ``GSVector2<int>`` at most cases.

Though I think ``std::atomic<bool>`` might be more readable at this case as not everyone might be aware of the atomic type definitions, not sure though.